pkg/reconciler/knativeserving/ingress/ingress.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ingress
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	mf "github.com/manifestival/manifestival"
+	"golang.org/x/mod/semver"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"knative.dev/operator/pkg/apis/operator/v1alpha1"
+	"knative.dev/operator/pkg/reconciler/common"
+)
+
+const providerLabel = "networking.knative.dev/ingress-provider"
+
+func ingressFilter(name string) mf.Predicate {
+	return func(u *unstructured.Unstructured) bool {
+		provider, hasLabel := u.GetLabels()[providerLabel]
+		if !hasLabel {
+			return true
```

Comment:
I might be wrong, but is this `return true` correct? If the resources have no `providerLabel`, operator manages it? :thinking:

Reply:
If the providerLabel is NOT found, the resource is then from the knative serving, so we need to keep it by returning true. :-)